cmd/service_run.go

```diff
@@ -398,12 +398,13 @@ func waitSig(ctx context.Context, cfunc func(), log *zap.Logger, pendingSigReque
 				p.CheckMark().SuccessText("Transaction approved").NextLine()
 
 				sentTx := <-sentTxs
+				log.Info("transaction sent", zap.Any("ID", sentTx.TxID), zap.Any("hash", sentTx.TxHash))
 				if sentTx.Error != nil {
 					p.BangMark().DangerText("Transaction failed! ").NextLine()
 					p.BangMark().DangerText("Error: ").DangerText(sentTx.Error.Error()).NextLine()
 					p.BangMark().DangerText("Details: ").DangerText(strings.Join(sentTx.ErrorDetails, " ,")).NextSection()
 				} else {
-					p.CheckMark().SuccessText("Transaction sent! ").NextLine()
+					p.CheckMark().SuccessText("Transaction sent! Hash:").SuccessText(sentTx.TxHash).NextLine()
```

Comment:
I think we can have better user-friendly message. What do you think about the following ?

Also, beware of the missing space when chaining formating function. Easy to forget.

```suggestion
					p.CheckMark().Text("Transaction with hash ").SuccessText(sentTx.TxHash).Text(" sent!").NextLine()
```